Back/Reciplastk.Gateway/Services/ShipmentTypeService.cs

```diff
@@ -0,0 +1,111 @@
+﻿using Reciplastk.Gateway.DataAccess;
+using Reciplastk.Gateway.Models;
+
+namespace Reciplastk.Gateway.Services
+{
+    public class ShipmentTypeService
+    {
+       private readonly ReciplastkContext db;
+       public ShipmentTypeService(ReciplastkContext db) 
+       {
+            this.db = db;
+       }
+        public HttpResponseModel ShowAllShipmentTypes()
+        {
+            var shipmentType = db.Shipmenttypes.Where(x => x.Isactive == true).ToList();
+            var response = new HttpResponseModel();
+            response.WasSuccessful = true;
+            response.Data = shipmentType;
+            return response;
+        }
+        public HttpResponseModel ShowShipmentType (int shipmenttypeid)
+        {
+            var response = new HttpResponseModel();
+            var shipmentType = db.Shipmenttypes.Where(x => x.Shipmenttypeid == shipmenttypeid && x.Isactive == true).FirstOrDefault();
```

Comment:
You can reuse the finShipmentById method.
```suggestion
            var shipmentType = FindShipmentById(shipmenttypeid );
```